themes/favrskovtheme/template/page.tpl.php

```diff
@@ -1 +1,12 @@
+<?php
+  // Added minimap.js only to article and news pages.
+  if ($node && (in_array($node->type, ['article', 'news']))) {
```

Comment:
Move code to init hook.
$node = menu_get_object();
to extract node. Variable will be empty on non-node pages.